util/update_repo.py

```diff
@@ -100,6 +101,8 @@ def build_versions(
     ]
     for asset in relevant_assets:
         is_v78 = "-solr78" in asset["name"]
+        if not (build_v78 == is_v78):
```

Comment:
```suggestion
        if build_v78 != is_v78:
```

Sieht einfacher aus :smile: